rust/src/ftp/response.rs

```diff
@@ -53,6 +54,9 @@ fn parse_response_line(input: &str) -> Option<FTPResponseLine> {
         _ => ("".to_string(), response_line), // No valid numeric code found
     };
 
+    let code_len = code.len();
+    let resp_len = response.len();
+
     // Convert response and code to C strings
     let c_code = CString::new(code).ok()?;
```

Comment:
We no longer need a 0-terminated CString ;-)

Reply:
The data is still access from C as through it were a string.

Reply:
Where ? I only see it accessed as a buffer...

Reply:
Will chane so a cstring is not used.